build/win_build_container.ps1

```diff
@@ -0,0 +1,41 @@
+<#
+    .SYNOPSIS
+        This script tests builds the New Relic Infrastructure Agent Windows container image
+#>
+param (
+    # Target base image:
+    [ValidateSet("1809", "1909", "2004" )]
```

Comment:
from what I understood on windows containers, this target should match to the gh runner os version . I think is 1809 currently.

Reply:
Yeah but in theory you could take the same script and assets and run it somewhere else to get a 2004 build. Also, if the GHA runners are updated to windows 20H2 we could then build the 1809 images on that machine as well.